packages/nextjs/app/builders/0x62CeF3Ca8b52a9C69a17236CA2c56Cdb7a383E8e/page.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+import { Github, Linkedin, Twitter } from "lucide-react";
+
+export default function BuilderProfile() {
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen bg-gradient-to-br from-base-200 to-base-300">
+      <div className="card w-96 bg-base-100 shadow-xl">
+        <figure className="px-10 pt-10">
+          <div className="avatar">
+            <div className="w-24 rounded-full ring ring-primary ring-offset-base-100 ring-offset-2">
+              <Image src="/superior.jpeg" width={96} height={96} alt="Builder Avatar" />
+            </div>
+          </div>
+        </figure>
+        <div className="card-body items-center text-center">
+          <h2 className="card-title">Samson Aderonmu</h2>
+          <p className="text-sm mt-2">
+            Passionate blockchain developer and Web3 enthusiast. Building the decentralized future, one smart contract
+            at a time.
+          </p>
+          <div className="card-actions justify-center mt-4">
+            <a
+              href="https://github.com/Superior212"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="btn btn-ghost btn-square"
+            >
+              <Github className="h-6 w-6" />
+            </a>
+            <a
+              href="https://x.com/Samsonaderonmu"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="btn btn-ghost btn-square"
+            >
+              <Twitter className="h-6 w-6" />
+            </a>
+            <a
+              href="https://www.linkedin.com/in/samsonaderonmu/e"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="btn btn-ghost btn-square"
+            >
+              <Linkedin className="h-6 w-6" />
+            </a>
```

Comment:
Since you are repeating yourself, you could create a social links array and use a map function.